torchvision/datasets/mnist.py

```diff
@@ -435,6 +435,74 @@ def get_int(b: bytes) -> int:
     return int(codecs.encode(b, 'hex'), 16)
 
 
+class MovingMNIST(VisionDataset):
+    """MovingMNIST"""
```

Comment:
Could you please a link (http://www.cs.toronto.edu/~nitish/unsupervised_video/) to the dataset like that 
```
`MNIST <http://yann.lecun.com/exdb/mnist/>`_ Dataset.
```
and define docstring `Args` etc as it is done for other datasets.